sql/identities/identities.go

```diff
@@ -62,6 +63,26 @@ func LoadMalfeasanceBlob(_ context.Context, db sql.Executor, nodeID []byte, blob
 	return err
 }
 
+func IterateMaliciousOps(
```

Comment:
This seems to be a bit of misnomer given that `identities` table may include not just malicious identities?
Also this func doesn't check for malicious identities in the same equivocation set

Reply:
This is not yet true, before ATX v2 goes live every entry in the `identities` table is malicious. I'm also planning on splitting the identities table for v1.8 when we have ATX v2 into an `identities` table that just keeps track of equivocation sets and a `malfeasance` table that keeps track of malicious identities. Work on this is still in progress here: https://github.com/spacemeshos/go-spacemesh/pull/6307

This is just an intermediary API that will be changed in the near future again 🙂